Client/Frontend/Browser/BrowserScrollController.swift

```diff
@@ -8,6 +8,11 @@ import SnapKit
 private let ToolbarBaseAnimationDuration: CGFloat = 0.3
 
 class BrowserScrollingController: NSObject {
+    enum ScrollDirection: Int {
+        case Up = 1
+        case Down = 2
+    }
```

Comment:
Any reason the `enum` has to be declared as an `Int` type here with explicit values?